test/transform-error/src/main.rs

```diff
@@ -0,0 +1,103 @@
+#![feature(box_syntax)]
+#![feature(asm)]
+#[macro_use]
+extern crate log;
+extern crate netbricks;
+extern crate simplelog;
+#[macro_use]
+extern crate failure;
+
+use self::nf::*;
+use log::Level;
+use netbricks::config::{basic_opts, read_matches};
+use netbricks::interface::*;
+use netbricks::operators::*;
+use netbricks::scheduler::*;
+use simplelog::{Config as SimpleConfig, LevelFilter, WriteLogger};
+use std::env;
+use std::fmt::Display;
+use std::fs::File as StdFile;
+use std::process;
+use std::sync::Arc;
+use std::thread;
+use std::time::Duration;
+mod nf;
+
+fn start_logger() {
+    WriteLogger::init(
+        LevelFilter::Warn,
+        SimpleConfig {
+            time: None,
+            level: Some(Level::Error),
+            target: Some(Level::Debug),
+            location: Some(Level::Trace),
+            time_format: None,
+        },
+        StdFile::create("test.log").unwrap(),
```

Comment:
so where does this file wind up? should we clean it up?

Reply:
@clinedome-work log files are auto ignored :).

Reply:
Also it’s in the test folder.